### DISCUSSION TITLE
[resolved] External Id w zdarzeniach dotyczących ofert

### DISCUSSION THREAD

**Discussion by pawelosajda** [allegro/allegro-api#4331] [Suggestions - resolved]
:bulb: **Sugestia**

Dzień dobry,
Przedstawię problem. Muszę synchronizować dużą liczbę aukcji wystawianych na 7 dni. Po wznowieniu aukcja otrzymuje nowy identyfikator. Śledzę zdarzenia dotyczące ofert i w zdarzeniach dotyczących aktywacji otrzymuję id nowej oferty. Wtedy muszę pobrać ofertę, by dowiedzieć się, jaką ma sygnaturę, jest to dodatkowy request. Czy byłaby możliwość przekazywania sygnatury oferty w strukturze OfferEventBaseOffer?

**Comment by MaciejFrackowiak** [comment]
Twoją sugestię przeniosłem na listę sugestii dot. rozwoju zasobów REST API.

**Comment by pkoziolandrobit** [comment]
Witam,
podpinam się pod prośbę. Oszczędziło by nam to setki tysięcy zapytań każdego dnia, a w niedługiej perspektywie pewnie 10 razy tyle.

**Comment by AureliuszBrussy** [comment]
@pkoziolandrobit dodaję Twój głos do sugestii.

**Comment by ppart** [comment]
Również podpinam się pod temat, dużo ułatwiło by to pracę.

**Comment by MaciejFrackowiak** [comment]
@ppart I Twój głos dodałem do listy.

**Comment by AureliuszBrussy** [comment]
Dodaliśmy dzisiaj pole external.id do dziennika zdarzeń w ofertach. [News dot. zmiany](https://github.com/allegro/allegro-api/issues/4684).
